testsuites/syncgateway/functional/topology_specific_tests/multiple_servers/conftest.py

```diff
@@ -62,6 +66,7 @@ def params_from_base_suite_setup(request):
     log_info("sa_installer_type: {}".format(sa_installer_type))
     log_info("delta_sync_enabled: {}".format(delta_sync_enabled))
     log_info("sg_platform: {}".format(sg_platform))
+    log_info("Delta_sync: {}".format(delta_sync_enabled))
```

Comment:
Duplicate info

Reply:
some conftest files are missing delta sync. Looks like I added to conftest files which already existing . Removed duplicate lines